sentry-resque/lib/sentry/resque.rb

```diff
@@ -32,7 +32,7 @@ def record(queue, worker, payload, &block)
 
               finish_transaction(transaction, 200)
             rescue Exception => exception
-              klass = payload['class'].constantize
+              klass = payload['class'].to_s.constantize
```

Comment:
If the class attribute is already a class, then we don't want to re-constantize it.

```suggestion
              klass = payload['class']
              if klass.is_a(String)
                klass = klass.constantize
              end
```